db/seeds.rb

```diff
@@ -5,3 +5,52 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+# users
+
+admin = [true, true, true, false, false, false, false, false, false, false, false, false, false, false, false]
```

Comment:
nur 15 items für admin?

Reply:
had only 15 in the beginning and realized that it might be too little given the various constraints on relevant groups; I changed it, but I think that it does not matter as it would still iterate 20 times over it.